### PR TITLE
fixes warnings, typos, added a header, etc

### PR DESCRIPTION
**Pull request by ajhunyady**

### CODE REVIEW

**Comment by ajhunyady** [comment]
@nicholastmosher, please review.

File: content/tutorials/rust/hello-world.md [hunk 1]
```diff
@@ -175,7 +187,7 @@ fn main() {
     let args: Vec<String> = std::env::args().collect();
     let args_slice: Vec<&str> = args.iter().map(|s| &**s).collect();
 
-    let result = match &*args_slice {
+    let _ = match &*args_slice {
```
Comment (nicholastmosher) [inline content/tutorials/rust/hello-world.md:190]: The `result` value was being used below to print errors if any occurred

Comment (ajhunyady) [reply content/tutorials/rust/hello-world.md:190]: My bad... I missed that part of code during copy/paste. I'll revert.

Comment (ajhunyady) [reply content/tutorials/rust/hello-world.md:190]: The 2nd section did not have it... hence the problem.

**Comment by nicholastmosher** [comment]
I just pushed a commit with some minor changes, otherwise it looks good to me 👍

**Review by nicholastmosher — APPROVED** [review]

**Comment by ajhunyady** [comment]
@nicholastmosher - Not sure if your Slack works as I didn't see a reply from you in a while. 

Also, can we also publish this code in the examples section of fluvio? Maybe `examples/01-hello-world`.
That way a fully working example is available to run.

**Pull request merged**